index.html

```diff
@@ -6,11 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header class="nav">
+        <nav class="nav-bar">
+            <ul>
+                Home
+            </ul>
+            <ul>
+                About us
+            </ul>
+            <ul>
+                Price List
+            </ul>
+            <ul>
+                Contact us
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="info">
+            <h1 class="title">
+                THE VEGAN CAKE SHOP
+            </h1>
+            <h2 class="tagline">
+                Here at the Vegan Cake shop we pride ourselves on tasty cakes that use 100% organic and vegan ingredients. We believe cakes do not need to upset the environment or harm any animals in the process. See our price list for all our pre-made delicious cakes or contact us for bespoke requests.
+            </h2>
+            <button class="order-now">
+                Order Now
+            </button>
+        </section>
+        <section class="cake">
+            <figure class="img1">
```

Comment:
A class typically means a group of things. It's a way to set rules at a group level, instead setting rules element by element.

Is there a class you think of that could apply to _all_ of these figures?

Reply:
i have removed the class and just used figure